search/indexing_api.py

```diff
@@ -432,27 +433,32 @@ def update_post(doc_id, post):
     )
 
 
-def index_items(serialize_bulk_items, object_type, ids):
+def index_items(documents, object_type):
     """
     Index items based on list of item ids
 
     Args:
-        serialize_bulk_items (callable): the function to serializer a list of objects by id
+        documents (iterable of dict): An iterable with ElasticSearch documents to index
         object_type (str): the ES object type
-        ids(list of int): List of item id's
     """
     conn = get_conn()
-    for alias in get_active_aliases(conn, [object_type]):
-        _, errors = bulk(
-            conn,
-            serialize_bulk_items(ids),
-            index=alias,
-            doc_type=GLOBAL_DOC_TYPE,
-            # Adjust chunk size from 500 depending on environment variable
-            chunk_size=settings.ELASTICSEARCH_INDEXING_CHUNK_SIZE,
-        )
-        if len(errors) > 0:
-            raise ReindexException(f"Error during bulk {object_type} insert: {errors}")
+    # bulk will also break an iterable into chunks. However we should do this here so that
+    # we can use the same documents when indexing to multiple aliases.
+    for chunk in chunks(
```

Comment:
@mbertrand I added this for loop to break the stream into chunks so we can use the same chunk for each alias in the inner for loop below.